src/main/java/io/github/pylonmc/pylon/base/content/magic/base/RuneApplicable.java

```diff
@@ -0,0 +1,34 @@
+package io.github.pylonmc.pylon.base.content.magic.base;
+
+import org.bukkit.event.player.PlayerDropItemEvent;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This interface defines what item can be applied with runes.
+ * Machines are not applicable by default.
```

Comment:
I'd rather list the default applicable items rather than just say "Machines are not applicable by default"